lightyear/src/server/connection.rs

```diff
@@ -462,6 +462,16 @@ impl Connection {
         }
     }
 
+    /// Return the latest estimate of rtt
+    pub fn rtt(&self) -> Duration {
```

Comment:
Maybe I should just make ping_manager public? but yes these are convenient

Reply:
slight preference towards a few nice fns in Connection, since there is internal stuff in pingmanager that games aren't concerned with. rtt(), jitter(), last_ping_time() or so. probably not many 🤷